tests/modeladmin/tests.py

```diff
@@ -666,6 +667,61 @@ def get_autocomplete_fields(self, request):
         finally:
             self.site.unregister(Band)
 
+    def test_get_deleted_objects_called(self):
+        """
+        ModelAdmin get_deleted_objects() method returns expected results from
+        django.contrib.admin.utils.get_deleted_objects.
+        """
+        mock_request = MockRequest()
+        mock_request.POST = {}
+        mock_request.META = {'SCRIPT_NAME': 'test'}
+        mock_request.user = User.objects.create_superuser(username='bob', email='bob@test.com', password='test')
+
+        ma = ModelAdmin(Band, self.site)
+
+        deletable_objects, model_count, perms_needed, protected = ma.get_deleted_objects(
+            [self.band], ma.opts, request, DEFAULT_DB_ALIAS
+        )
+        self.assertEqual(deletable_objects, ['Band: The Doors'])
+        self.assertEqual(model_count, {'bands': 1})
+        self.assertEqual(perms_needed, set())
+        self.assertEqual(protected, [])
+
+    def test_overriding_get_deleted_objects(self):
```

Comment:
This test belongs with the tests for the delete action in `tests/admin_views/test_actions.py`.